resources/prometheus/prometheus-rules.yaml

```diff
@@ -539,11 +539,24 @@ spec:
             )
           record: central:success_rate10m
 
+        - expr: |
+            probe_success{app="rhacs", endpoint="central-ui"}
+          record: central:ui_health
+
+        - expr: |
+            probe_success{app="rhacs", endpoint="central-data"}
```

Comment:
What is the difference between central health and central readiness probe?

Reply:
If you mean the **pod** readiness probe, then the difference is the following:
1. It checks the pod internally within the cluster, but blackbox-exporter checks also the public endpoint (route)
2. It doesn't check TLS certificates.

If you mean something else, please specify.

Reply:
Yes that's what I meant. Could you please add this explanation as a comment as well?